src/valkey-benchmark.c

```diff
@@ -2122,7 +2173,7 @@ int main(int argc, char **argv) {
         if (test_is_selected("mset")) {
             const char *cmd_argv[21];
             cmd_argv[0] = "MSET";
-            sds key_placeholder = sdscatprintf(sdsnew(""), "key%s:__rand_int__", tag);
+            sds key_placeholder = sdscatprintf(sdsnew(""), "key%s:%s", tag, BENCHMARK_PLACEHOLDERS[0]);
             for (i = 1; i < 21; i += 2) {
                 cmd_argv[i] = key_placeholder;
                 cmd_argv[i + 1] = data;
```

Comment:
The 10 keys in MGET need to be different keys when running without --sequential and with --sequential too.

It seems safest we should keep the old behavior for `__rand_int__` and introduce the new behavior only for the new placeholders.